spec/integration/store/Preferences/Appearance.spec.js

```diff
@@ -8,23 +8,27 @@ import DefaultFonts from '@/utils/fonts'
 import Appearance from '@/store/Preferences/Appearance'
 import { ipcMain } from '~/spec/mock/electron'
 
-const state = {
-  appearance: {
-    theme: Theme.Light.key,
-    fontSize: 14,
-    displayNameStyle: DisplayStyle.DisplayNameAndUsername.value,
-    timeFormat: TimeFormat.Absolute.value,
-    customThemeColor: LightTheme,
-    font: DefaultFonts[0]
-  },
-  fonts: []
+const state = () => {
+  return {
+    appearance: {
+      theme: Theme.Light.key,
+      fontSize: 14,
+      displayNameStyle: DisplayStyle.DisplayNameAndUsername.value,
+      timeFormat: TimeFormat.Absolute.value,
+      customThemeColor: LightTheme,
+      font: DefaultFonts[0]
+    },
+    fonts: []
+  }
 }
 
-const Preferences = {
-  namespaced: true,
-  state: state,
-  actions: Appearance.actions,
-  mutations: Appearance.mutations
+const initStore =() => {
```

Comment:
__[ESLint]__ Error - Infix operators must be spaced. (space-infix-ops)

<sub>posted by Sider</sub>